examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>are we using <code>data_path</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>not using, removed. </p>